hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/TestBlockEvictionOnRegionMovement.java

```diff
@@ -82,6 +83,7 @@ public void setup() throws Exception {
     conf.set("hbase.bucketcache.persistent.path", testDir + "/bucket.persistence");
     conf.setLong(CacheConfig.BUCKETCACHE_PERSIST_INTERVAL_KEY, 100);
     conf.setBoolean(CacheConfig.EVICT_BLOCKS_ON_CLOSE_KEY, true);
+    conf.setBoolean(CACHE_BLOCKS_ON_WRITE_KEY, true);
```

Comment:
[nit] why do we need to add the cache blocks on write here?  does anyone change the test behavior? I cannot find any other edits in this file.

Reply:
Yes, this is not related to this throttling change, but I found out this test was flakey, with the assertion from line 104 failing sometimes.  Most of the times, it will pass, as once we flush the newly added data on line 95, we'll eventually open the newly created file and run a prefetch for it. But this happens in the background, and sometimes, the prefetch might not have cached any blocks when we get to line 104 of this test. So, if we want to make sure blocks are cached at that point, we should have blockOnWrite set to true.